package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -188,4 +190,72 @@ function M.log(message, verbose)
 	posix_syslog.syslog(posix_syslog.LOG_INFO, message)
 end
 
+M.subprocess = {}
+
+M.subprocess.DEVNULL = -1
+M.subprocess.INHERIT = nil
+M.subprocess.PIPE = 1
+
+-- Execute a program found using command PATH search, like the shell.
+-- Return the pid, as well as the I/O streams as pipes or nil on error.
+function M.subprocess.popen(policies, path, ...)
+	local childfds = {}
+	local parentfds = {}
+
+	for fd, policy in pairs(policies) do
+		if policy==M.subprocess.PIPE then
```

Comment:
some inconsistency in spacing.

Reply:
Indeed, thanks.
I think i got them all in 2502b2c, right?